media/js/respiration/forestgraph.js

```diff
@@ -3,10 +3,10 @@ function ForestGraphData() {
     this.species = {};
     this.colors = ['#ff1f81', '#a21764', '#8ab438', '#999999', '#3a5b87', '#00c0c7', '#c070f0', '#ff8000', '#00ff00'];
     /* adding things here to experiment */
-    this.showError
-    this.clearError
-    this.errorHighlight
-    this.clearHighlight
+    this.showError;
```

Comment:
Ideally, experiments should not appear on master unless absolutely necessary. Can these bits be removed?